assets/css/style.css

```diff
@@ -4483,6 +4483,22 @@ div.media-sidebar a.edit-attachment {
   background-size: 25px 25px;
   z-index: 999;
 }
+@media (max-width: 480px){
```

Comment:
Style.css changes will be overwritten after running the grunt command.
Move responsive related css code to response.less and store related css to store.less

Reply:
Also exclude this file from this pr

Reply:
Done Bhai